src/coprocessor/dag/expr/builtin_miscellaneous.rs

```diff
@@ -12,6 +14,82 @@ const IPV4_LENGTH: usize = 4;
 const PREFIX_COMPAT: [u8; 12] = [0x00; 12];
 
 impl ScalarFunc {
+    pub fn int_any_value(&self, ctx: &mut EvalContext, row: &[Datum]) -> Result<Option<i64>> {
+        if self.children.is_empty() {
```

Comment:
I'd prefer writing these without panic possibility:

```rust
if let Some(child) = self.children.first() {
    child.eval_int(ctx, row)
} else {
    Ok(None)
}
```

or maybe through combinators (but it is harder to read):

```rust
self.children.first().map_or(Ok(None), |child| child.eval_int(ctx, row))
```